communication-service/index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const cors = require("cors");
+const http = require("http");
+
+const app = express();
+
+app.use(express.urlencoded({ extended: true }));
+app.use(express.json());
+app.use(cors()); // config cors so that front-end can use
+app.options("*", cors());
+
```

Comment:
Can you add a get endpoint with path `/`.

This is for the ingress to check if the server is live when we deploy on GCP.

Reply:
fixed.